apisix/plugin.lua

```diff
@@ -1234,6 +1236,26 @@ function _M.run_plugin(phase, plugins, api_ctx)
     return api_ctx, plugin_run
 end
 
+function _M.set_plugins_meta_parent(plugins, parent)
+    if not plugins then
+        return
+    end
+    for _, plugin_conf in pairs(plugins) do
+        if not plugin_conf._meta then
+            plugin_conf._meta = {}
+        end
+        if not plugin_conf._meta.parent then
+            local mt_table = getmetatable(plugin_conf._meta)
+            if mt_table then
+                mt_table.parent = parent
```

Comment:
https://github.com/apache/apisix/pull/12426

this PR has beed merge

so the parent should be a resource path, do you think?

Reply:
yes you're correct. I missed this, let me correct.

Reply:
the healthcheck_manager module is tied only to manage the checkers on the upstream configuration. I don't know if we can use that mechanism here because here we are creating custom healthchecker using the checks configuration in the plugin configuration.